lib/services/access_list.go

```diff
@@ -174,6 +178,12 @@ type AccessListMembers interface {
 	DeleteAllAccessListMembers(ctx context.Context) error
 }
 
+type AccessListRecursiveMembersGetter interface {
+	ListAccessListMembers(ctx context.Context, accessListName string, pageSize int, pageToken string) (members []*accesslist.AccessListMember, nextToken string, err error)
+	GetAccessListMember(ctx context.Context, accessList string, memberName string) (*accesslist.AccessListMember, error)
+	GetAccessList(context.Context, string) (*accesslist.AccessList, error)
+}
```

Comment:
How the `Recursive` logic applie to the interface methods ? 

I would assume that if I have `i AccessListRecursiveMembersGetter` and call i.GetAccessListMember the `GetAccessListMember` method on recursive interface will travel all nested access list and return the all members by visiting nested access list chain.